src/components/EditableField/EditableField.tsx

```diff
@@ -414,7 +427,26 @@ export class EditableField extends React.Component<
     }
   }
 
-  renderInputFields() {
+  renderAddButton = () => {
+    const { disabled } = this.props
+    const { fieldValue, multipleValuesEnabled } = this.state
+
+    const isLastValueEmpty = fieldValue[fieldValue.length - 1].value === ''
+    const isSingleAndEmpty = fieldValue.length === 1 && isLastValueEmpty
+
+    return multipleValuesEnabled && !isSingleAndEmpty && !disabled ? (
+      <AddButtonUI
+        className="EditableField_addButton"
+        type="button"
+        onClick={this.handleAddValue}
+        disabled={isLastValueEmpty}
+      >
+        <Icon name={ACTION_ICONS['plus']} size="20" />
```

Comment:
You can just use `ACTIONS_ICONS.plus` here.

Reply:
Ah yeah, thanks I removed the suffix and now I can change that 👍